cmd/performance-profile-creator/cmd/root.go

```diff
@@ -39,7 +39,8 @@ const (
 )
 
 var (
-	validTMPolicyValues = []string{"single-numa-node", "best-effort", "restricted", "none", ""}
+	validTMPolicyValues        = []string{"single-numa-node", "best-effort", "restricted", "none", ""}
+	validPowerConsumptionModes = []string{"cstate", "no-state", "idle-poll"}
```

Comment:
@MarSik @fromanirh are the names clear enough? 
What we have in design is:
Power consumption mode
   Default (cstate/pstate sleep level limit to 1)
   Ignore power consumption (don’t use c-states to save power)
   No power saving  - (idle=pool))

Maybe we can have
   "limit-cstates" - default
   "ignore"
    "no-power-saving"

Can you come up with better names?

Also do you remember maybe what kernel args *exactly* we should use for it? We can delay this part for when we'll send a PR for it, but maybe you have the answer.

Reply:
My take: I'd abstract the power saving setting from the implementation, hence I'd go with something like

- default: maps to your default
- performance: maps to your "ignore power consumption" (is it actually ignoring?)
- low-latency: maps to "no power saving"

or something like it. The docs should also further clarify that the higher we go (low-latency is higher than performance) the more power the system will consume.